### DISCUSSION TITLE
Regional formats date&time

### DISCUSSION THREAD

**Discussion by fib1985** [firefly-iii/firefly-iii#9585] [Firefly III] [answered]
Hi everyone,

I have a small problem with firefly. I recently switched from docker desktop (windows) to fedora (linux) and everything works fine (even backup and restore). I have set the regional format correctly in the OS and env files but firefly doesn't want to know about reading the date in the correct format ddMMYYY 24H. Is there any way to force it somewhere else? thx

ps. I also tried setting the os display language to English but it doesn't change.

![Screenshot From 2024-12-27 21-01-06](https://github.com/user-attachments/assets/56cfd5c3-cfa4-493e-8353-42d6ec1b8251)
![Screenshot From 2024-12-27 21-01-30](https://github.com/user-attachments/assets/acac7c51-ea20-4790-91d9-61372751d09a)
![Screenshot From 2024-12-27 15-17-36](https://github.com/user-attachments/assets/54f04dbe-5da5-49a2-ba48-106bf528a9cf)
![Screenshot From 2024-12-27 15-19-28](https://github.com/user-attachments/assets/657634b0-e18c-4799-ae7e-8baf9565564a)

**Comment by JC5** [comment, marked as answer]
Hi, thanks for starting a discussion!

This gets asked fairly regularly and is in fact a browser setting. I use an HTML field "datetime-local" and it follows whatever your browser is set to.

**Reply by fib1985** [reply]
thanks! I just switched my browser language <3
